src/CodeGen_Hexagon.cpp

```diff
@@ -531,6 +534,8 @@ void CodeGen_Hexagon::init_module() {
 
         // Broadcasts
         { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplatw), u32v1,  "splat.w", {u32} },
+        { IPICK(is_128B, Intrinsic::hexagon_V6_lvsplatb), u8v1,   "splat.b", {u8}  },   // v62 - Broadcasts
```

Comment:
I realized that I was wrong, these need flags and to be only enabled on v62 targets. The problem is that these already have an implementation in the runtime modules (hvx_64.ll and hvx_128.ll) that emulates these instructions. They have weak linkage, so what you have here will work on v62 targets (it will call this definition over the one in the runtime modules), but it will also generate calls to them on v60 targets.

The easiest fix would be to add a v62 flag to HvxIntrinsic, and skip defining them if the target doesn't support v62. And as long as we're doing it for these, we might as well add that flag for the rest so as to be more consistent/less confusing.